app/assets/javascripts/discourse-i18n/src/index.js

```diff
@@ -36,6 +33,13 @@ export class I18n {
     return this.locale || this.defaultLocale;
   }
 
+  get normalizedLocale() {
+    if (this.currentLocale() === "pt") {
```

Comment:
This is a special case from `make-plural`. `pt` is Brazilian Portuguese, `pt-PT` is the one for Portugal. But our locales don’t match this, for us `pt` is Portugal Portuguese and `pt-BR` Brazilian Portuguese.

Reply:
It sounds like this normalization is specific to pluralization, and wouldn't make sense to use elsewhere in our codebase? If so, I wonder whether a more specific name would be better? Like "pluralizationNormalizedLocale"?

Perhaps it should even be a private getter like `#pluralizationNormalizedLocale`?

Reply:
I’ve renamed it and left it public as I can’t unit test it otherwise. (I tried to access a private field in various ways but it doesn’t seem possible :thinking:)